libs/components/indicators/testing/src/public-api.ts

```diff
@@ -1,11 +1,17 @@
+export * from './alert/alert-harness';
+export * from './alert/alert-harness-filters';
+
+export * from './key-info/key-info-harness';
+export * from './key-info/key-info-harness-filters';
+
+export * from './label/label-harness';
+export * from './label/label-harness-filters';
+
 export * from './tokens/token-harness';
 export * from './tokens/token-harness-filters';
 export * from './tokens/tokens-harness';
 export * from './tokens/tokens-harness-filters';
 
-export * from './alert-fixture';
-export * from './label-fixture';
-export * from './wait-fixture';
-
-export * from './alert/alert-harness';
-export * from './alert/alert-harness-filters';
+export * from './old-fixtures/alert-fixture';
```

Comment:
Did we decide on the word "old"? I'd personally prefer a folder named "deprecated", but we can discuss as a team offline if needed.

Reply:
I went with old because they aren't all deprecated yet. Feels kind of weird to name a folder deprecated when not all of its contents actually are

Reply:
Since they aren't all deprecated, I just moved the fixtures back to the root folder